transport/tchannel/handler.go

```diff
@@ -94,6 +94,11 @@ func (h handler) handle(ctx context.Context, call inboundCall) {
 	responseWriter := newResponseWriter(call.Response(), call.Format())
 
 	err := h.callHandler(ctx, call, responseWriter)
+	if _codeToTChannelCode[yarpcerrors.FromError(err).Code()] == tchannel.ErrCodeBusy {
```

Comment:
Maybe just do `if yarpcerrors.ErrorCode(err) == yarpcerrors.CodeResourceExhausted`

Reply:
We have `.ErrorCode()` marked as [deprecated](https://godoc.org/go.uber.org/yarpc/yarpcerrors#ErrorCode) now, but I'll change this to compare against `yarpcerrors.CodeResourceExhausted` on my next update.